### PR TITLE
Various cleanup for DPDK repo

### PR DESCRIPTION
**Pull request by pprindeville**
Drop `kmod-vfio*` as they're already upstreamed.

Include `libnuma` as a `libdpdk` dependency.

Make `numactl` buildable on ARM64.

### CODE REVIEW

File: packages/numactl/Makefile [hunk 1]
```diff
@@ -32,13 +32,13 @@ define Package/numactl
    CATEGORY:=Utilities
    TITLE:=Control NUMA policy for processes or shared memory
    URL:=https://github.com/numactl/numactl
-   DEPENDS:=@TARGET_x86_64 +libpthread +libstdcpp +libc +librt +libnuma
+   DEPENDS:=@(x86_64||aarch64) +libpthread +libstdcpp +libc +librt +libnuma
```
Comment (pprindeville) [inline packages/numactl/Makefile:35]: Don't know if this should include `arm` as well.  It's not clear from the `numactl` `README.md` that 32-bit processors are adequately supported.

Comment (zokl) [reply packages/numactl/Makefile:35]: I will try it on x86 and arm but in 2 weeks.

File: packages/kmod-vfio/Makefile
Comment (zokl) [inline packages/kmod-vfio/Makefile:1]: Openwrt 21.02 does not have this module. It could be in this repo without required dependency in dpdk.

**Review by zokl — APPROVED** [review]
The patch is OK.

File: packages/numactl/Makefile (hunk 1 above)
Comment (zokl) [reply packages/numactl/Makefile:35]: ARM is obsolete. aarch64 is OK.

**Pull request merged**